Project 01 - GoodAsOldPhones/GoodAsOldPhones/ProductsTableViewController.swift

```diff
@@ -9,32 +9,31 @@ import UIKit
 
 class ProductsTableViewController: UITableViewController {
   fileprivate var products: [Product]?
-  fileprivate let identifer = "productCell"
```

Comment:
In my opinion, this is not a good idea to remove the identifier. Removing this will lead to hard code in tableview datasource function, which is not a good code style.